### ISSUE TITLE
a few (custom) application to category mappings are changed randomly

### ISSUE THREAD

**Issue by 612wong** [ntop/ntopng#6699]
**Environment**:
* OS name: RHEL
* OS version: 7.9
* Architecture: x86_64
* ntopng version/revision: ntopng Enterprise M v.5.3.220612 (CentOS Linux release 7.9.2009 (Core))

**What happened**:
I created a few custom applications (e.g. VoIP_A, VoIP_C) and grouped them into VoIP category. They might change to another category such as Cloud. I also created a custom application influxDB and grouped it into Database category. Then it was moved into another category. This happened to a few other custom application. I found it only when I browsed through the application list under each category.

**What did you expect to happen?**
I didn't expect the application to category could be changed itself.

**How did you reproduce it?**
It happened randomly. When I tried to fix an application list under a category, which may have changed the others.

**Debug Information**:

**Comment by lucaderi** [comment]
Not sure I understand what you did Please post your configuration and explain how to reproduce the results

**Comment by 612wong** [comment]
I just fixed my VoIP application and Cloud category and then I found a few application-to-category mappings are changed.

For example, IT_WindowsServer was mapped to 'system' cateogry. it is now mapped to 'network' category.
<img width="892" alt="Screen Shot 2022-06-14 at 16 40 58" src="https://user-images.githubusercontent.com/58540705/173519898-4e8f5a1b-49fe-496b-8422-d364013d5a1e.png">

For example, Kafka_GCP was mapped to 'cloud' category, it is now mapped to 'system' category.
<img width="794" alt="Screen Shot 2022-06-14 at 16 42 06" src="https://user-images.githubusercontent.com/58540705/173520333-2caa47c5-18c5-4d66-b920-539e049fc8b1.png">

For example, Kafka was mapped to 'streaming' category, it is now mapped to 'user custom category 1'.
<img width="809" alt="Screen Shot 2022-06-14 at 16 42 49" src="https://user-images.githubusercontent.com/58540705/173520480-c4999383-2911-4a3c-ba33-8b3765ff1e06.png">

Is it okay I send you the whole configuration privately?
